### ISSUE TITLE
关于utils.py中dataloader错误的纠正

### ISSUE THREAD

**Issue by zhanghaok** [xuyouqian/Bert-Ner-Demo#1]
大佬您好，看了你的代码很赞。
不过今天在学习的时候，发现一点小问题
utils.py的第223行此处`loader = DataLoader(ner, batch_size=config.batch_size, shuffle=True)`代码有误。
应该改为`loader = DataLoader(ner, batch_size=config.batch_size, shuffle=True,collate_fn=collate_fn)`

**Comment by xuyouqian** [comment]
感谢大佬哈 那个github的代码我没调试 或者你可以看我kaggle上的链接 那个是跑的通有效果的哈 https://www.kaggle.com/xuyouqian/bert-nested-ner



在 2022-02-21 19:36:25，"Zhanghaok" ***@***.***> 写道：

大佬您好，看了你的代码很赞。
不过今天在学习的时候，发现一点小问题
utils.py的第223行此处loader = DataLoader(ner, batch_size=config.batch_size, shuffle=True)代码有误。
应该改为loader = DataLoader(ner, batch_size=config.batch_size, shuffle=True,**collate_fn=collate_fn**)

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by zhanghaok** [comment]
好的，谢谢大佬，我参考一下kaggle，github上那代码训练的特别慢




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "xuyouqian/Bert-Ner-Demo"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年2月22日(星期二) 中午1:44
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [xuyouqian/Bert-Ner-Demo] 关于utils.py中dataloader错误的纠正 (Issue #1)





 感谢大佬哈 那个github的代码我没调试 或者你可以看我kaggle上的链接 那个是跑的通有效果的哈 https://www.kaggle.com/xuyouqian/bert-nested-ner
 
 
 
 在 2022-02-21 19:36:25，"Zhanghaok" ***@***.***&gt; 写道：
 
 大佬您好，看了你的代码很赞。
 不过今天在学习的时候，发现一点小问题
 utils.py的第223行此处loader = DataLoader(ner, batch_size=config.batch_size, shuffle=True)代码有误。
 应该改为loader = DataLoader(ner, batch_size=config.batch_size, shuffle=True,**collate_fn=collate_fn**)
 
 —
 Reply to this email directly, view it on GitHub, or unsubscribe.
 Triage notifications on the go with GitHub Mobile for iOS or Android.
 You are receiving this because you are subscribed to this thread.Message ID: ***@***.***&gt; 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
